src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -301,6 +301,7 @@ public EarlyStoppingRankingMetric EarlyStoppingMetric
             public Options()
             {
                 EarlyStoppingMetric = EarlyStoppingRankingMetric.NdcgAt1; // Use L1 by default.
+                RowGroupColumnName = "GroupId"; // Use GroupId as default for ranking options.
```

Comment:
**Suggestion**: Use `DefaultColumnNames.GroupId` instead, also the one in LightGbm

Reply:
Good point, I will change that.